### PR TITLE
Write "No providers" when no providers on detail views

### PR DESCRIPTION
**Pull request by Primdahl26**
image

### CODE REVIEW

**Comment by Primdahl26** [comment]
Added no providers for `tv` & `movie` if there are no providers as a badge.

image

This also cleans up the providers logic, so it will do 2 rows if there are more than 5 providers, or else it will keep 1 row of providers. This also works on mobile. When testing a good id with 5 providers: `/tv/44217`, and a good provider with more than 5: `/tv/12225`

Lastly this also makes the card on top be bordered when in mobile view.

image

File: frontend/src/components/details/top_card.svelte [hunk 1]
```diff
@@ -44,17 +44,32 @@
...
-                </div>
-            {/each}
-        </div>
+                {/each}
+            </div>
+        {:else if !providers.length}
+            <div class="pt-5">
+                <div class="badge badge-lg">No providers</div>
+            </div>
+        {:else}
```
Comment (AndreasPB) [inline frontend/src/components/details/top_card.svelte:62]: The if/elif/else order hurts my brain, mby 1. not, 2. more, 3. less? or reverse

**Review by AndreasPB — APPROVED** [review]
:shipit: LGTM with the one change

File: frontend/src/components/details/top_card.svelte (hunk 1 above)
Comment (Primdahl26) [reply frontend/src/components/details/top_card.svelte]: Done.

**Pull request merged**